src/main/content/_assets/js/docs.js

```diff
@@ -53,7 +53,7 @@ function selectDocInToc() {
     else {
         pathName = `/docs/${location.pathname.split('/')[2]}`
     }
-    let selectedFile = `${pathName}/ref/general` + currentHref.substring(currentHref.lastIndexOf('/'));
+    let selectedFile = `${pathName}/ref/general/${location.pathname.split('/')[4]}` + currentHref.substring(currentHref.lastIndexOf('/'));
```

Comment:
can you add a comment on what `${location.pathname.split('/')[4]}` is expected to be? and a comment for `currentHref.substring(currentHref.lastIndexOf('/')` too